app/src/main/java/com/asfoundation/wallet/topup/TopUpFragmentPresenter.kt

```diff
@@ -165,6 +170,23 @@ class TopUpFragmentPresenter(
       .subscribe()
   }
 
+  private fun updateRefundDisclaimerValue() {
+    disposables.add(
+      getShowRefundDisclaimerCodeUseCase().subscribeOn(networkScheduler).observeOn(viewScheduler)
+        .doOnSuccess {
+          if (it.showRefundDisclaimer == 1) {
```

Comment:
This values can go into a enum (same for topup, IAP and onboarding), so that is easier to understand what each value mean